doc/dftb+/manual/dftbp.tex

```diff
@@ -6488,10 +6489,20 @@ \subsection{TRA.DAT}
 the single particle transitions.
 
 \subsection{NACV.DAT}
+\label{subsec:nacv.dat}
 \index{NACV.DAT}
 
-TD-(LC)-DFTB non-adiabatic coupling vectors between states. First the indices of the
-two states are written, then the coupling vector follows. 
+Contains the TD-(LC)-DFTB non-adiabatic coupling vectors between
+many-body states, $$d^\xi_{mn} = \mel{\Psi_m}{\dv{x_\xi}}{\Psi_n}.$$
```

Comment:
Should be 
$$d^\xi_{mn} = \mel{\Psi_m}{\dv{\xi}}{\Psi_n}.$$

Reply:
Thanks, now changed.